app/src/main/java/com.example.weatherapp/data/services/response/MainResponse.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.weatherapp.data.services.response
+
+import com.google.gson.annotations.SerializedName
+
+data class MainResponse(
+    @SerializedName("temp") val temp: String,
+    @SerializedName("feels_like") val feels_like: String,
```

Comment:
use the common naming standards instead of **feels_like** use **feelsLike**

Reply:
check all the variables on this class

Reply:
Changed name of variables on model and response data classes to common naming standards.